src/entities/FactorOperationalMemory.jl

```diff
@@ -181,6 +153,8 @@ mutable struct CommonConvWrapper{ T<:AbstractFactor,
   _gradients::G
   # type used for cache
   dummyCache::CT
+  #
+  fullvariables::Vector{DFGVariable}
```

Comment:
yeah, cleanup of CCW after FMd and CPT are removed.  That should start to help performance, allocations, dynamic dispatch etc.